modules/ROOT/pages/kubernetes/deploy-cluster.adoc

```diff
@@ -111,7 +111,7 @@ Import node::
 --
 [source,console]
 ----
-kubectl create secret generic sgw-config-import --from-file sgw-config-import.json
+kubectl create secret generic sgw-config-import -f sgw-config-import.json
```

Comment:
This one is actually right! It creates a secret from a file, it's just the deployment ones that were wrong.